backend/src/migrations/019_2426_remove_orphaned_reviews_responses.ts

```diff
@@ -0,0 +1,41 @@
+import ReleaseModel from '../models/Release.js'
+import { ReviewDoc } from '../models/Review.js'
+import { UserInterface } from '../models/User.js'
+import { removeResponses } from '../services/response.js'
+import { removeReleaseReviews } from '../services/review.js'
+
+/**
+ * This isn't really used but needed for consistency
+ */
+const migrationUser: UserInterface = {
+  dn: 'migration',
+}
+
+export async function up() {
+  // Find all releases have already been marked as deleted
+  const deletedReleases = await ReleaseModel.find({ deleted: true }).lean()
```

Comment:
We should remove `lean` as it's the semver is failing validation when this object is returned as a POJO. We also need to consider access request reviews as well as releases